packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx

```diff
@@ -122,6 +124,37 @@ export const IntercomCard: React.FC<
         />
       );
       break;
+
+    case 'settingView':
+      view = (
+        <>
+          <SettingHeader classNames={classNames?.NotifiIntercomSettingHeader} />
+          <div className={'NotifiIntercomCardSettingContent__container'}>
+            <div
+              className={clsx(
+                'NotifiIntercomCard__subtitle2',
+                classNames?.subtitle2,
+              )}
+            >
+              {companySupportDescription}
+            </div>
+            <NotifiIntercomFTUNotificationTargetSection
```

Comment:
Can you please add the icons in each target? (ex. Mail icon, cell phone icon, Telegram icon, etc.)

<img width="311" alt="Screen Shot 2022-11-11 at 9 47 51 AM" src="https://user-images.githubusercontent.com/108922885/201399206-476703ac-0f19-4bd9-9cca-e3519bffba03.png">

Reply:
attached the updated screenshot in the description box.